backend/db/migrate/20220817011608_create_property_assessments.rb

```diff
@@ -1,8 +1,22 @@
 class CreatePropertyAssessments < ActiveRecord::Migration[7.0]
+  def up
+    execute <<-SQL
+      CREATE TYPE condition AS ENUM ('not_set', 'poor', 'fair', 'average', 'good', 'excellent');
+    SQL
+  end
+  # Type for condition values for parcels, rooms, etc.
+  enum condition:  { not_set: 'not_set', poor: 'poor', fair: 'fair', average:'average',  good: 'good', excellent: 'excellent' }
```

Comment:
Let's use the Rails enum as shown in http://api.rubyonrails.org/classes/ActiveRecord/Enum.html for now, so we don't have to worry about pulling through the native Postgres type.